test/e2e/e2e_test.go

```diff
@@ -99,6 +99,15 @@ func TestMain(m *testing.M) {
 	flag.StringVar(&globalOpts.HOInstallationOptions.PlatformMonitoring, "e2e.platform-monitoring", "All", "The option for enabling platform cluster monitoring when installing the HyperShift Operator. Valid values are: None, OperatorOnly, All. This is a HyperShift Operator installation option")
 	flag.BoolVar(&globalOpts.RunUpgradeTest, "upgrade.run-tests", false, "Run HyperShift Operator upgrade test")
 
+	//ext OIDC configuration
```

Comment:
@heliubj18 , nit, many occurrences of "//xxx" without a space between. How about updating them to be "// xxxx"?

Reply:
done